setup.cfg

```diff
@@ -1,6 +1,7 @@
 [flake8]
 min-version = 3.6
 accept-encodings = utf-8
+max-line-length = 88
```

Comment:
black is configured for a line-length of 80 in [pyproject.toml](https://github.com/beetbox/beets/pull/4654/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R2)